### ISSUE TITLE
baud rate for xl-320

### ISSUE THREAD

**Issue by jjehl** [poppy-project/pypot#226]
The table in conversion.py for baudrate is not suitable for xl-320 : 
dynamixelBaudrates = {
    1: 1000000.0,
    3: 500000.0,
    4: 400000.0,
    7: 250000.0,
    9: 200000.0,
    16: 117647.1,
    34: 57600.0,
    103: 19230.8,
    207: 9615.4,
    250: 2250000.0,
    251: 2500000.0,
    252: 3000000.0,
}

It is probably why io.change_baudrate doesn't work.

The table for xl-320 should be : 
0: 9600, 1:57600, 2:115200, 3:1Mbps

Do you think adding a new table with model key is a good way to fix the issue ?

```
def dxl_to_baudrate(value, model):
    return dynamixelBaudrates[model][value]
```

**Comment by show0k** [comment]
Good catch, this is obviously a bug and yes, we should have a new table depending on the model. 
From [XL320 doc](http://support.robotis.com/en/product/actuator/dynamixel_x/xl_series/xl-320.htm#Actuator_Address_04):
![screen shot 2017-04-07 at 14 27 34](https://cloud.githubusercontent.com/assets/3891499/24800015/8b99dbfc-1b9e-11e7-8f71-2397fdb84c12.png)

Feel free to do a PR.

**Comment by show0k** [comment]
Fixed by #273
